comms/src/inbound_message_service/inbound_message_service.rs

```diff
@@ -37,134 +35,112 @@ use crate::{
 };
 use log::*;
 use serde::{de::DeserializeOwned, Serialize};
-#[cfg(test)]
-use std::sync::mpsc::{sync_channel, Receiver};
 use std::{
     sync::{mpsc::SyncSender, Arc},
     thread::JoinHandle,
 };
 
 const LOG_TARGET: &'static str = "comms::inbound_message_service";
 
-/// The maximum number of processing worker threads that will be created by the InboundMessageService
-const MAX_INBOUND_MSG_PROCESSING_WORKERS: u8 = 2; // TODO read this from config
+#[derive(Clone, Copy)]
+pub struct InboundMessageServiceConfig {
+    /// Timeout used for receiving messages from the message queue
+    pub worker_timeout_in_ms: u32,
```

Comment:
Nits: Why not use Duration? Then you don't need the `in_ms` suffix.

Also, note this timeout is u32 and the one below is u64. (They could both possibly be Duration types though).